docs/section4/4-10.md

```diff
@@ -0,0 +1,38 @@
+# ASHVAC – Rule 4-10
+
```

Comment:
Did we agree to split occupied from unoccupied?

Reply:
Yup.